src/interfaces/models.ts

```diff
@@ -217,7 +220,7 @@ export interface RequestNameI {
   consent_words: '' | string[]
   conflict1: string
   conflict1_num: string
-  request_type_cd?: string
+  request_type_cd?: NameType
```

Comment:
I doubt if the request_type_cd can only be ex-pro name type here?

Reply:
You're probably right but it's not possible to confirm this because of the type-less assignments, eg,
```
  data['request_type_cd'] = <something>
```

I have reverted these declarations back to strings in the next commit.